eq-author/src/components/CommentsPanel/CommentSection.js

```diff
@@ -10,34 +10,22 @@ import EditReply from "./EditReply";
 
 import {
   CommentAddSection,
-  CommentHeaderContainer,
-  AvatarWrapper,
-  AvatarOuter,
-  AvatarInner,
-  NameWrapper,
-  FlexLabel,
   DateField,
-  EditButton,
-  DeleteComment,
   StyledTextArea,
   CommentsDiv,
   DateWrapper,
   CommentFooterContainer,
-  SaveButton,
 } from "./index";
 
-const ReplyButton = styled(Button)`
-  padding: 0.3em 0.8em 0.4em;
-  display: ${props => (props.isHidden ? "none" : "block")};
-`;
+import CommentHeader from "./CommentHeader";
 
 const StyledAccordion = styled(CommentAccordion)`
   background-color: blue;
 `;
```

Comment:
Whilst you're here, could you change this colour to use our colour library please?